test/libgit2.jl

```diff
@@ -1910,8 +1921,11 @@ mktempdir() do dir
                             deserialize(f)
                         end
                         @test err.code == LibGit2.Error.ERROR
-                        @test err.msg == "Invalid Content-Type: text/plain"
+                        @test err.msg == "invalid Content-Type: text/plain"
```

Comment:
These tests only run on Linux so they should always be using the specified version. I could and the conditional if it is still preferable